src/bitbucket/BitbucketAPI.ts

```diff
@@ -31,8 +33,9 @@ export class BitbucketAPI {
     Logger.info('Merging pull request', {
       namespace: 'bitbucket:api:mergePullRequest',
       pullRequestId,
-      requestId,
+      landRequestId,
       targetBranch,
+      landRequestStatus,
```

Comment:
so we will display the LandRequestStatus sent to Splunk
* awaiting merge
* ...

Reply:
Exactly, when just logging `landRequest` we couldn't see the current state.